packages/truffle-compile/test/test_supplier.js

```diff
@@ -125,7 +125,7 @@ describe("CompilerSupplier", function() {
 
       const compilerCacheDirectory = path.resolve(
         Config.getTruffleDataDirectory(),
-        "compilers"
+        "compilers/node_modules"
```

Comment:
Is it worth de-duplicating this? Since this path join happens in two places now. Just a thought.

Reply:
I like this test since it sanity checks if the supplier is actually working. I don't think the test would work as it is intended w/o updating the compilerCacheDirectory path to be what BYOC is now using.